package.json

```diff
@@ -6,7 +6,8 @@
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "test": "jest",
     "tsc": "tsc -p . --noEmit",
-    "lint": "eslint --fix --ext .mjs,.js,.jsx,.ts,.tsx ."
+    "lint": "eslint --fix --ext .mjs,.js,.jsx,.ts,.tsx .",
+    "overlay-config": "./scripts/download-config.sh $PWD"
```

Comment:
@textileio/team once this is merged, you can run `yarn overlay-config` at any time to pull down the latest dev config files